HLshell/mycmd_1.py

```diff
@@ -100,13 +113,13 @@ def do_exit(self, par):
         return True
 
     def help_exit(self):
-        print('type q or qqq to leave')
+        print('type exit or qqq to leave')
 
     def default(self, inp):
-        if inp == 'q' or bool(match(inp,'qqq+')):
-            return self.do_exit()
+        if inp == 'exit' or inp == 'qqq':
```

Comment:
I prefer typing 'q' or many q to leave. If program crashes, it may receive many q.
The final decision depends on you, choose one and delete the other.

Reply:
put the bool match back in